### PR TITLE
Job model

### PR DESCRIPTION
**Pull request by AshkenYariv**

### CODE REVIEW

**Review by shayakrach — APPROVED** [review]

File: jobs/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Jobs',
```
Comment (assafg95) [inline jobs/migrations/0001_initial.py:18]: the model should be named Job and not Jobs. Django adds automatically the plural

**Comment by yuvalk** [comment]
seems like this PR includes #21 
can you plz properly split them (dont use the same origin branch, will make it easier for you)

**Comment by yuvalk** [comment]
> seems like this PR includes #21 can you plz properly split them (dont use the same origin branch, will make it easier for you)

I can help with the git technicalities if needed

**Comment by AshkenYariv** [comment]
@yuvalk The PR's are related to one another, jobs uses a foreign key from users thus they cant be from different origins. Can we close PR 21 and thus PR 23 would be a PR with 3 commits?

**Comment by AshkenYariv** [comment]
Moving all the change to PR#21

**Pull request closed without merging**